app/javascript/availability/index.js

```diff
@@ -406,7 +406,7 @@ const availability = {
 
     generateCallNumber(holding) {
         // Do not display call number for on loan items
-        return (holding.locationID === 'ILLEND') ? "" : holding.callNumber;
+        return (holding.locationID === "ILLEND") ? "" : holding.callNumber;
```

Comment:
was this a lint thing? single versus double quotes?

Reply:
ohhh this was just me trying to attempt for consistency but then I see that there are single and double in multiple places 😬